### PR TITLE
Add initial issue template

### PR DESCRIPTION
**Pull request by willingc**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  you are facing. Please be as thorough as you are able to provide details on
+  the issue.
+
+**How to reproduce the issue**
+
+**What you expected to happen**
+
+**What actually happens**
+
+**Share what version of JupyterHub you are using**
```
Comment (parente) [inline .github/issue_template.md]: Can we ask the reporter to run `jupyter troubleshoot` too if he/she is knowledgeable about the command line?

Comment (willingc) [inline .github/issue_template.md]: Yes, we should 👍 Slipped my mind.

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Hi! Thanks for using JupyterHub.
+
+If you are reporting an issue with JupyterHub:
+
+- Please use the GitHub issue search feature to check if your issue has been
+  asked already. If it has, please add your comments to the existing issue.
+
+- Please fill out the details below to help us troubleshoot the issue that
+  you are facing. Please be as thorough as you are able to provide details on
+  the issue.
```
Comment (minrk) [inline .github/issue_template.md]: Can we add a 'where applicable' or similar qualifier here? I think some can be intimidated by having several fields that they may not be able to answer; I know I have been. I know the 'as thorough as you are able' covers this, but I'd like to be extra explicit about it.

**Comment by parente** [comment]
Thanks @willingc. 🎈

**Pull request merged**